psalm-baseline.xml

```diff
@@ -873,7 +873,8 @@
     <DocblockTypeContradiction occurrences="1">
       <code>null === $this-&gt;xpath</code>
     </DocblockTypeContradiction>
-    <MissingConstructor occurrences="14">
+    <MissingConstructor occurrences="15">
```

Comment:
You seem to be already aware of it, but we really should have near-zero additions to these baseline files

Reply:
Do you want me to create a seperate PR to fix this bug?